src/Eurofurence.App.Server.Web/Identity/ApiKeyAuthenticationOptions.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Authentication;
+
+namespace Eurofurence.App.Server.Web.Identity;
+
+public class ApiKeyAuthenticationOptions : AuthenticationSchemeOptions
+{
+    public IList<ApiKeyOptions> ApiKeys { get; set; }
+    public class ApiKeyOptions {
+        public string Key { get; set;}
+        public string PrincipalName { get; set;}
+        public DateTime ValidUntil { get; set;}
```

Comment:
Probably not an issue but that's a local time

Reply:
Thanks for the hint; right now having this be precise enough on a day-level is sufficient though. (=